src/Stratis.Bitcoin/Consensus/Validators/BlockValidator.cs

```diff
@@ -165,6 +176,29 @@ public void StartPartialValidation(ChainedHeaderBlock chainedHeaderBlock, OnPart
             this.logger.LogTrace("(-)");
         }
 
+        /// <inheritdoc />
+        public async Task<PartialValidationResult> ValidateAsync(Block block, ChainedHeader chainedHeader)
+        {
+            this.logger.LogTrace("({0}:'{1}')", nameof(block), block.GetHash());
+
+            var chainedHeaderBlock = new ChainedHeaderBlock(block, chainedHeader);
+            var validationContext = new ValidationContext { Block = block, ChainTipToExtand = chainedHeader };
+
+            await this.consensusRules.PartialValidationAsync(validationContext).ConfigureAwait(false);
+
+            var partialValidationResult = new PartialValidationResult
+            {
+                BanDurationSeconds = validationContext.BanDurationSeconds,
+                BanReason = validationContext.Error != null ? $"Invalid block received: {validationContext.Error.Message}" : string.Empty,
+                ChainedHeaderBlock = validationContext.Error == null ? chainedHeaderBlock : null,
+                Error = validationContext.Error,
+                Succeeded = validationContext.Error == null
+            };
+
+            this.logger.LogTrace("(-)");
```

Comment:
Return value here again

Reply:
Just added the validation result's `Succeeded` value?